pypit/artrace.py

```diff
@@ -814,11 +854,11 @@ def trace_object(slf, det, sciframe, varframe, crmask, trim=2,
     # Convert left object trace
     for o in range(nobj):
         trccopy[:, o] = trcfunc[:, o] - cval[o] + objl[o]/(npix-1.0)
-    trobjl = ofst + (diff-triml-trimr)*trccopy
+    #trobjl = ofst + (diff-triml-trimr)*trccopy
```

Comment:
I suspect this problem arises because of the mixup between `triml`, `trimr`, and `xedge`.

Reply:
Can we put a TODO in here too, with the same general mention as above? Just so that the commented out code doesn't get deleted somewhere down the line by mistake.

Reply:
Added, although I don't think trobjl nor trobjr are needed
anymore (there were only for QA and that is being handled
differently now).